RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/factory/EventType.kt

```diff
@@ -13,5 +13,6 @@ enum class EventType(val text: String)
 	PAGE_VIEW("PageView"),
 	KEEP_ALIVE("KeepAlive"),
 	VIDEO("VidEvent"),
-	ERROR("ErrEvent"),
+    AUREUS_IMPRESSION_EVENT("AureusImpressionEvent"),
```

Comment:
tu jakis problem chyba z tabulacja?

Reply:
Ciekawe - nie wiem czy nie jest to kwestia wyświetlania przez GitHub - w ide wyglada poprawnie...